src/Spryker/Zed/RabbitMq/Business/Model/Queue/Queue.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq\Business\Model\Queue;
+
+use Psr\Log\LoggerInterface;
+use Spryker\Client\Queue\Model\Adapter\AdapterInterface;
+
+class Queue implements QueueInterface
```

Comment:
Why do you need such class `Queue` and it has only deleting/purging functionality inside, maybe you can change the name to something else?